clients/privacy-center/packages/fides-consent/README.md

```diff
@@ -51,10 +51,36 @@ For example, the default configuration includes a consent option for advertising
 When a user visits a page that includes `fides-consent.js` with this configuration, the value of
 `Fides.consent.data_sales` will be set to `true` by default.
 
-If the user visits the Privacy Center and removes their consent for advertising, this choice is
+If the user visits the Privacy Center and removes their consent for `advertising`, this choice is
 saved in their browser. Subsequent visits to pages that include `fides-consent.js` will have 
 `Fides.consent.data_sales` set to `false`.
 
+Multiple data uses can refer to the same cookie key. A user must consent to _all_ of those data uses
+for the cookie key to be true. For example, with the default configuration:
+
+```json
+{
+  "consent": {
+    "consentOptions": [
+      {
+        "fidesDataUseKey": "advertising.first_party",
+        "name": "Email Marketing",
+        "default": true,
+        "cookieKeys": ["tracking"]
+      },
+      {
+        "fidesDataUseKey": "improve",
+        "name": "Product Analytics",
+        "default": true,
+        "cookieKeys": ["tracking"]
+      }
+    ]
+  }
+}
+```
+
+By default, `Fides.consent.tracking` will be set to `true`. If the user removes their consent for 
+`advertising.first_party`, `improve`, or both, then `Fides.consent.tracking` will be set to `false`.
```

Comment:
@mfbrown I'd like your opinion on this. I've added a cookie key called "tracking" that (I think) fits with Shopify and Meta's vague cookie consent descriptions. Their options are not granular at all -- it's basically "identify the user for any reason that isn't an essential function." Check out the links I've added in this readme to see if you agree. 

(Note that this is just the _default_ config. We can change it later.)

Reply:
@mfbrown 🧵 
> This makes sense to me in general. Since we are just shipping the defaults, do you think it makes sense to create cookie keys that are even more explicit? Something like meta_tracking and shopify_tracking since we can have multiple keys on a data use?
> I know it's a bit repetitive, but the very obvious naming might make it easier for someone unfamiliar with the project to parse.

IMO adding platform names to the cookie keys* might make it harder to understand because that would make it look like integrations require those prefixes or even that they're automatically enabled by using the prefix. 

\* I'm thinking of renaming "cookie keys" to "consent types"

Reply:
https://github.com/ethyca/fides/issues/2346